src/components/ClickablePicture/ClickablePicture.js

```diff
@@ -0,0 +1,28 @@
+import './ClickablePicture.css';
+import React, {Component} from 'react';
+
+class ClickablePicture extends Component {
+    state = {
+        picture: this.props.img
+    };
+
+    toggleImage(){
+        if(this.state.picture === this.props.img){
```

Comment:
este if no está mal, pero lo puedes simplificar con un ternario dentro del setState:

```suggestion
        this.setState({
            picture: this.state.picture === this.props.img ? this.props.imgClicked : this.props.img
        })
```